chart/Chart.yaml

```diff
@@ -25,3 +25,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     version: 5.5.12
     condition: fluentd.enabled
+  - name: metrics-server
+    repository: https://kubernetes-sigs.github.io/metrics-server
+    version: 3.10.0
+    condition: hpa.enabled
```

Comment:
What if a client has the metrics server already installed?

Reply:
> What if a client has the metrics server already installed?

I'm hoping that should still be okay, but something we can test anyway. We should document this clearly as well.